app/src/main/java/org/mozilla/fenix/home/pocket/PocketCategoriesViewHolder.kt

```diff
@@ -55,12 +59,48 @@ class PocketCategoriesViewHolder(
         val categoriesSelections = components.appStore
             .observeAsComposableState { state -> state.pocketStoriesCategoriesSelections }.value
 
+        val wallpaperState = components.appStore
+            .observeAsComposableState { state -> state.wallpaperState }.value ?: WallpaperState.default
+        val wallpaperIsDefault = Wallpaper.nameIsDefault(wallpaperState.currentWallpaper.name)
+
+        val selectedBackgroundColor: Color?
+        val unselectedBackgroundColor: Color?
+        val selectedTextColor: Color?
+        val unselectedTextColor: Color?
+        when {
```

Comment:
Does this need to be a `when` seems like it could just be an `if/else`?

Reply:
I'll update this. It was originally 3 conditions, but when I simplified it, I did not condense the `when` into an `if/else`